### ISSUE TITLE
Crashes before finishing

### ISSUE THREAD

**Issue by dlewand691** [kush-agra/Dynamic_Walls-app#4]
Have tried running 1.0.1 and it gets as far as asking to allow for location then crashes with no warning. Have tried several times with no luck.

**Comment by djlythium** [comment]
Same thing is happening to me. Asks to use dynamic wallpaper, allow access to Terminal, then Terminal pops up for a second, then the whole deal quits.

**Comment by kush-agra** [comment]
make sure your location services are on, I'm guessing it's because of that

also make sure the mac is connected to wifi, nneds it for the location and that one api call to get the sunrise/set times

**Comment by djlythium** [comment]
Yep, all of that was on. No dice.
    On Tuesday, September 25, 2018, 7:31:19 AM PDT, kush-agra <notifications@github.com> wrote:  
 
 
make sure your location services are on, I'm guessing it's because of that

also make sure the mac is connected to wifi, nneds it for the location and that one api call to get the sunrise/set times

**Comment by kush-agra** [comment]
I just tried a fresh install and it works for me so I'll have to check deeper, you're on mojave? it asks for 2 permissions on mojave, one for terminal and other for system events, did you allow both of those? if it never asked for the second one just click the app icon again in the dock and it pops up, seems to be something related to mojave

**Comment by djlythium** [comment]
Yep, on Mohave, and allowed both of those permission-asks. I had light mode on, though it was at night, so expected the app to switch the system to dark mode, but nothing happened. Also, after allowing permission, the app just disappears from the dock. Is that normal behavior?
Thanks for your help!
    On Tuesday, September 25, 2018, 9:42:43 PM PDT, kush-agra <notifications@github.com> wrote:  
 
 
I just tried a fresh install and it works for me so I'll have to check deeper, you're on mojave? it asks for 2 permissions on mojave, one for terminal and other for system events, did you allow both of those? if it never asked for the second one just click the app icon again in the dock and it pops up, seems to be something related to mojave

**Comment by kush-agra** [comment]
yes the app and terminal closing automatically is how it is intended to be, are you choosing a correctly formatted wallpapers folder? it should automatically switch to dark mode, I'll test if it still works properly on mojave the next chance I get

**Comment by h1r0ss** [comment]
Same here, I think. After having asked for terminal and location permissions and asking "Use dynamic system theme?" it quits. Could you describe what should happen after the process? Maybe it happens and I just can't figure it out. Thanks!
